tests/manager/api/overdrive/test_api.py

```diff
@@ -2028,45 +2028,6 @@ def test__refresh_patron_oauth_token_failure(
                 credential, patron, "a pin"
             )
 
-    def test__refresh_patron_oauth_token_palace_context(
-        self, overdrive_api_fixture: OverdriveAPIFixture
-    ):
```

Comment:
Is this already being tested elsewhere? I would've expected to see a test related to the collection context being removed.

Reply:
>  I would've expected to see a test related to the collection context being removed.
@tdilauro I'm not sure I understand 

This test, I think, covers the case you are talking about. If not can you expand on your question a bit?
https://github.com/ThePalaceProject/circulation/blob/e644bbbb82cbdb1ff0994e4bb3c756c1060b4a13/tests/manager/api/overdrive/test_api.py#L1957-L1995

Reply:
Yes, that exactly answers my question. Thanks!